src/transforms/spark-jobs/geojson_to_db/main.py

```diff
@@ -0,0 +1,126 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+import argparse
+import json
+import psycopg2
+from azure.storage.blob import BlobServiceClient
+from notebookutils import mssparkutils
+from pyspark.sql import SparkSession
+
+# Collect args
+parser = argparse.ArgumentParser(description='Arguements required to run geojson to postgres script')
+parser.add_argument('--storage_account_name', type=str, required=True, help='Name of the storage account name where the input data resides')
+parser.add_argument('--container_name', type=str, required=False, help='Container that host the data file')
+parser.add_argument('--folder_path', type=str, required=False, help='Path of the folder relative to the container')
+
+parser.add_argument('--db_username', type=str, required=False, help='Username of the database for authentication')
+parser.add_argument('--db_host', default=None, required=True, help='Full host name of the database server')
+parser.add_argument('--db_name', default=None, required=False, help='Name of the database')
+
+parser.add_argument('--key_vault_name', type=str, required=True, help='Name of the Key Vault that stores the secrets')
+parser.add_argument('--storage_account_key_secret_name', type=str, required=True, help='Name of the secret in the Key Vault that stores storage account key')
+parser.add_argument('--linked_service_name', type=str, required=True, help='Name of the Linked Service for the Key Vault')
+parser.add_argument('--db_password_secret_name', type=str, required=True, help='Name of the secret in the Key Vault that stores the database password')
+
+#Parse Args
+args = parser.parse_args()
+
+def save_blob(file_name: str,file_content):
+
+    # Get full path to the file
+    download_file_path = file_name
+
+    # for nested blobs, create local path as well!
+    # os.makedirs(os.path.dirname(download_file_path), exist_ok=True)
+    with open(download_file_path, "wb") as file:
+      file.write(file_content)
+
+def download_file_from_storage_account(storage_account_name: str, storage_account_key: str, container_name: str, folder_path: str,  file_name: str):
+
+    storage_account_url = f'https://{storage_account_name}.blob.core.windows.net'
```

Comment:
This is specific to the public cloud. This may differ for Government and other Clouds.